site/styles/_meta-detail.scss

```diff
@@ -240,6 +240,19 @@ s72-element-switcher,
       animation: fadein 2s;
     }
   }
+
+  .s72-pricing-button-container {
```

Comment:
Bex is wrong about this... you should delete it IMO.

- It isn't more visually appealing... it looks cramped.
- It isn't consistent... these buttons should look the same wherever they are.
- You can't use `flex` here unless you also override to unset `gap` (because Safari sucks)... but doing it that way is CSSpaghetti... you should just use `grid`.
- This override should live in `_cta-buttons.scss`... one thing should live in one place.  (though in this case I prefer if it didn't exist at all).

Reply:
Asked Bex and have reverted css, this pr is now just translations